docs/station_listing.ipynb

```diff
@@ -8,7 +8,9 @@
    "source": [
     "# Station listing\n",
     "\n",
-    "This site provides an overview of available solar radiation monitoring stations worldwide and metadata of the stations."
+    "This site provides an overview of available solar radiation monitoring stations worldwide and metadata of the stations.\n",
+    "\n",
+    "The table is also available to download in CSV form: {download}`station-listing.csv`"
```

Comment:
```suggestion
    "The table is also available to download in CSV form: {download}`solar-stations-listing.csv`"
```

Reply:
It would be nice to have "SolarStations" as part of the downloaded filename. The above is just a suggestion. Maybe even include the ".org" part so that people know where it came from?

Reply:
`SolarStationsOrg-station-listing.csv`?  I'll update this PR to whatever name you like :)